bin/looker-resources-index/yarn.lock

```diff
@@ -0,0 +1,13 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
From the description:

> This folder contains data (in code) about Looker-related resources and scripts to maintain and repackage that data for publication.

It doesn't appear that this is production code, or even intended to be installed? @jkaster can confirm. If this is the case, there is no need for a lockfile.

If, indeed, this is meant to be installed _and_ is used in production, we should add it to this folder (`bin/looker-resources-index`) to the `workspaces` array in the root `package.json` file.

Either way, we should not add a lockfile here.

Reply:
I reverted that now

Reply:
The only things consumed from this folder are the json files